extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -36,7 +36,7 @@ public function index() {
 			'heroImageUrl' => $this->getHeroImageUrl(),
 			'inviteFriendsText' => $this->msg( 'communitypage-invite-friends' )->plain(),
 			'headerWelcomeMsg' => $this->msg( 'communitypage-tasks-header-welcome', $wgWikiTopic ?? $wgSitename )->parse(),
-			'adminWelcomeMsg' => $this->msg( 'communitypage-admin-welcome-message' )->text(),
+			'subheaderWelcomeMsg' => $this->msg( 'communitypage-subheader-welcome' )->text(),
```

Comment:
I wasn't changing that, but good catch, looks like it is not included here: https://github.com/Wikia/app/pull/11322/files